packages/sandbox/package.json

```diff
@@ -8,14 +8,15 @@
     "@testing-library/user-event": "^12.6.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-scripts": "^4.0.2"
+    "react-scripts": "^4.0.2",
+    "typescript": "^4.6.2"
```

Comment:
please define `typescript` as a dev dependency. This [post](https://dev.to/philippurwoko/should-i-install-typescript-globally-or-at-devdepencendy-468o) can help us to understand the reason

Reply:
sure :D